modules/imaging_browser/php/imaging_session_controlpanel.class.inc

```diff
@@ -159,16 +159,12 @@ class Imaging_Session_ControlPanel
                                            'true'  => 'True',
                                            'false' => 'False',
                                           );
-        $query    = "SELECT MRIQCStatus, MRIQCPending,".
-                    " MRICaveat FROM session WHERE ID=:sid";
-        $qcstatus = $DB->pselectRow(
-            $query,
-            array("sid" => $this->sessionID)
-        );
 
-        $subjectData['mriqcstatus']  = $qcstatus['MRIQCStatus'];
-        $subjectData['mriqcpending'] = $qcstatus['MRIQCPending'];
-        $subjectData['mricaveat']    = $qcstatus['MRICaveat'];
+        $qcstatus = $timePoint->getImagingQC();
```

Comment:
This is unrelated to the API but I wanted to show usage of the add `getImagingQC` function in TimePoint outside of the API.